hathor/manager.py

```diff
@@ -475,7 +477,7 @@ def _initialize_components(self) -> None:
 
         # Check if all checkpoints in database are ok
         my_best_height = self.tx_storage.get_height_best_block()
-        if checkpoint_heights:
+        if checkpoint_heights and False:
```

Comment:
Skipping verification?

Reply:
This is debug code that wasn't removed.

Reply:
I guess you have already removed it. Just double checking.

Reply:
Yes.